test/ui-test/schemaIsSetTest.ts

```diff
@@ -0,0 +1,51 @@
+import path = require('path');
+import os = require('os');
+import { WebDriver, VSBrowser, Key, InputBox, TextEditor, StatusBar, By } from 'vscode-extension-tester';
+
+/**
+ * @author Zbynek Cervinka <zcervink@redhat.com>
+ */
+export function schemaIsSetTest(): void {
+  describe('Verify that the right JSON schema has been selected', () => {
+    it('The right JSON schema has been selected', async function () {
+      this.timeout(30000);
+
+      const driver: WebDriver = VSBrowser.instance.driver;
+      await driver.actions().sendKeys(Key.F1).perform();
+
+      let input = await InputBox.create();
+      await input.setText('>new file');
+      await input.confirm();
+      await input.confirm();
+
+      await driver.actions().sendKeys(Key.chord(TextEditor.ctlKey, 's')).perform();
+      input = await InputBox.create();
+      await input.setText('~/kustomization.yaml');
+      await input.confirm();
+
+      (await VSBrowser.instance.driver.wait(() => {
```

Comment:
Lambda function can be async since you are calling asynchronous function in it.

Reply:
I do not understand, It seems ok to me. What should I change there?

Reply:
fixed